src/main/views/macro/feesTable.njk

```diff
@@ -0,0 +1,29 @@
+{% macro feesTable(label, rows, noHearingFee=false) %}
+  <details class="notice">
+    <summary>{{ t(label) }}</summary>
+
+    <div class="panel">
+      <p>
+        {{ t('The fee is based on the amount you’re claiming, including any interest.')}}
+      </p>
+      <table>
+        <thead>
+        <tr>
+          <th scope="col">{{ t('Claim amount') }}</th>
+          <th scope="col">{{ t('Claim fee') }}</th>
+          {% if noHearingFee %}<th scope="col">{{ t('Hearing fee') }}</th>{% endif %}
```

Comment:
err, if no hearing fee then add hearing fee?

Reply:
how about `includeHearingFee` and then default it to true above? not sure which option is more common though

Reply:
updated to use showHearingFee

Reply:
it's only used in two places so it's either/or at the moment although I expect showing just the claim fees will be more than hearing centre fees